frontend/src/utils/format_column_data.ts

```
@@ -20,255 +20,152 @@ export type Edge = {
-      child.group_radius;
+    const top_level = this.adj_list.get(this.root);
 
-    return [x, y];
-  }
+    if (!top_level) {
+      return rows;
+    }
 
-  /*
+    const que: string[][] = [[]];
+    let curr_q_idx: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)